src/projects/document_project.rst

```diff
@@ -46,36 +46,82 @@ be released with the same cadence as the code and it will always be possible
 to trace a particular version of the documentation to a particular version 
 of the code documented. 
 
-Documentation must be placed in a ``docs`` folder in the upper level of the 
-repository structure. 
+Free text documentation must be placed in a ``docs`` folder in the upper level of the
+repository structure. Sphinx generates automatically extracted files under this folder as well, wherever there are docstrings in the code.
 
 Using sphinx
 ============
 
 Documentation must be realised using the `sphinx <http://www.sphinx-doc.org>`_  
 package and `Restructured Text <http://docutils.sourceforge.net/rst.html>`_ .
 SKA provides a predefined sphinx template for this purpose in the 
-`SKA python skeleton <https://github.com/ska-telescope/ska-python-skeleton>`_ project. 
+`SKA Python skeleton <https://github.com/ska-telescope/ska-python-skeleton>`_ project.
 Every project shall use the same ``docs`` folder as a starting point for assembling their 
 own documentation.
 
 Sphinx can be used to generate text documents such as this portal, but it also provides 
-capabilities to automatically extract and parse code documentation.  
+capabilities to automatically extract and parse code documentation, or docstrings.
+
+Docstrings
```

Comment:
should refer to python coding guidelines at http://developer.skatelescope.org/en/latest/development/python-codeguide.html 
Also check for consistency and maintain one single point where this is defined.